### PR TITLE
Trying different ways of deploying

### PR DESCRIPTION
**Pull request by AlexGagne**

### CODE REVIEW

**Review by TomasMorton — APPROVED** [review]
IMO cleaner

File: .circleci/config.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@
  version: 2.1
  workflows:
    pull-request:
-     unless:
+     equal:
        equal: ["trunk", << pipeline.git.branch >>]
```
Comment (TomasMorton) [inline .circleci/config.yml]: Won't this run on master now?

Comment (AlexGagne) [reply .circleci/config.yml]: Yes, this is on purpose. I can't figure out the deploy to ECS

Comment (AlexGagne) [reply .circleci/config.yml]: I'm going to revert when I'm done testing lol

Comment (TomasMorton) [reply .circleci/config.yml]: Ah. Usually my team won't switch the entire workflow as it's higher risk, rather temporarily insert the needed step into the PR workflow. But nothing is gonna blow up right now.

**Pull request merged**